plugins/action_change_status/src/incrementStage.test.ts

```diff
@@ -0,0 +1,6 @@
+// Placeholder file for tests that should be written
+
+// Test cases:
+// - new application => set status to Draft
+// - existing application => change status
+// - existing application => try and set status to illegal value
```

Comment:
I haven't realised until I saw another `incrementStage.test` file, this should be renamed to proper test file : )